### DISCUSSION TITLE
How does aarch64 support coredump?

### DISCUSSION THREAD

**Discussion by peng6662001** [cloud-hypervisor/cloud-hypervisor#5978] [Q&A]
I built clh with `cargo build --feature guest_debug`, then tested coredump with the following commands on Ampere Altra
```
./target/debug/ch-remote --api-socket /tmp/cloud-hypervisor.sock pause
./target/debug/ch-remote --api-socket /tmp/cloud-hypervisor.sock coredump file:///tmp/snapshot/ch.dmp
./target/debug/ch-remote --api-socket /tmp/cloud-hypervisor.sock resume
```
It returned an error:`NotFound: NotFound`

I found code in /vmm/src/vm.rs#L306
```
    #[cfg(all(target_arch = "x86_64", feature = "guest_debug"))]
    #[error("Error coredumping VM: {0:?}")]
    Coredump(GuestDebuggableError),
```
Is Coredump only supported by x86_64?
How does aarch64 support coredump?

**Comment by peng6662001** [comment]
@jongwu Does ARM plan to support coredump on CH?

**Reply by jongwu** [reply]
@michael2012z -, Any idea?

**Comment by michael2012z** [comment]
I haven't ever tested coredump on arm64. I will look into the gap and be back later.
